maintainers/maintainer-list.nix

```diff
@@ -3237,6 +3237,12 @@
     githubId = 10528737;
     name = "Severin Fürbringer";
   };
+  fufexan = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add fufexan``.

Reply:
Done.